### DISCUSSION TITLE
Monterey beta warnings

### DISCUSSION THREAD

**Discussion by OldSubSailor** [Homebrew/discussions#2818] [Everyday usage]
Howdy,
Two days ago, I deleted and reinstalled Homebrew. I am using an intel iMac and Monterey 12.1. As Homebrew was installing, I saw some warnings about Monterey, Homebrew and beta. Can anyone tell me what that was all about.
There were no issues with Homebrew or the installed packages that I had, I was changing things, updating so-to-speak to see how things would work with a program called Ortho4XP. Some new users were having a lot of trouble with the install and run, so I thought that I would try a "clean" install in an effort to see what was happening. 
Right now, the only problem seems to be with GDAL (again).

**Comment by gromgit** [comment]
What's the output of `brew config` and `brew doctor`?

**Reply by OldSubSailor** [reply]
Howdy gromgit, 
I did not pay much attention to brew doctor seeing as how they pretty much say to ignore the warnings, but I should have posted eh?
[Brewdoctorconfig.txt](https://github.com/Homebrew/discussions/files/7906731/Brewdoctorconfig.txt)
numpy is a python pip install.  And just a reminder, other than USGS, things are working for the Ortho4XP app. I was wondering about why Homebrew was issuing warnings about beta status for Monterey seeing as how Monterey has been out so long. 
If I need to, I can uninstall Homebrew again and try to capture those warnings. I meant to, but my 75 year old brain forgot it as I got caught up in playing with Ortho.

**Reply by gromgit** [reply]
Hmmm, nothing seems off. Which command(s) triggers the beta warning, and what's the exact wording of that warning?

**Comment by OldSubSailor** [comment]
Hummm, I apologize I may not have been clear in my first post. It was when I reinstalled Homebrew and as typical of a Homebrew install, it was among the dozens of lines of text that is generated. There was something about 'unsupported beta' Monterey among the crap ton of lines that are generated.  

I uninstalled Homebrew and attempted to recreate the messages. I think that it was caused by having a mismatched and out of date command line interface installed. After I had gone through the earlier processes, I was also required to update and _then_ install a new command line interface.   

So this time I did not get any of the beta warnings. 
[Homebrew and Gdal Install.txt](https://github.com/Homebrew/discussions/files/7915044/Homebrew.and.Gdal.Install.txt)
